templates/visuals/rvisual/src/visual.ts

```diff
@@ -73,8 +79,44 @@ module powerbi.extensibility.visual {
             this.imageDiv.style.width = finalViewport.width + 'px';
         }
 
+        /**
+         * This function gets called from the update function above. You should read the new values of the properties into 
+         * your settings object so you will be able to use the new value in the enumerateObjectInstances function below.
+         * 
+         * Below there is a code snippet for a case where you want to expose a single property called "lineColor" from the object called "settings"
```

Comment:
* Below is a code snippet demonstrating how to expose a single property called "lineColor" from the object called "settings"
 +